portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -21,26 +21,52 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
+import java.util.Arrays;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  Gson gson = new Gson();  
+private static Gson gson = new Gson();  
```

Comment:
I believe you should be able to make gson final also.

Reply:
Ok